tsl/test/expected/hypercore_create.out

```diff
@@ -921,3 +921,74 @@ select * from rides order by pickup_datetime;
  745233436676 | Fri Jan 01 00:00:03 2016 PST | Fri Jan 01 00:11:14 2016 PST |               1 |          6.00 | -73.947151184082031 | 40.791046142578125 |         1 | -73.920768737792969 | 40.865577697753906 |            2 |           9 |   0.5 |     0.5 |          0 |            0 |                   0.3 |         19.3
 (3 rows)
 
+-- All these are valid methods to set the default
+show timescaledb.default_hypercore_use_access_method;
+ timescaledb.default_hypercore_use_access_method 
+-------------------------------------------------
+ none
+(1 row)
+
+set timescaledb.default_hypercore_use_access_method to on;
+set timescaledb.default_hypercore_use_access_method to off;
+set timescaledb.default_hypercore_use_access_method to true;
+set timescaledb.default_hypercore_use_access_method to false;
+set timescaledb.default_hypercore_use_access_method to yes;
+set timescaledb.default_hypercore_use_access_method to no;
+set timescaledb.default_hypercore_use_access_method to 0;
+set timescaledb.default_hypercore_use_access_method to 1;
+show timescaledb.default_hypercore_use_access_method;
+ timescaledb.default_hypercore_use_access_method 
+-------------------------------------------------
+ on
+(1 row)
+
+-- This should generate an error and not change the value
+\set ON_ERROR_STOP 0
+\set VERBOSITY default
+set timescaledb.default_hypercore_use_access_method to magic;
+ERROR:  invalid value for parameter "timescaledb.default_hypercore_use_access_method": "magic"
+HINT:  Available values: none, on, off.
+show timescaledb.default_hypercore_use_access_method;
+ timescaledb.default_hypercore_use_access_method 
+-------------------------------------------------
+ on
+(1 row)
+
+\set VERBOSITY terse
+\set ON_ERROR_STOP 1
+-- This should unset the value
+reset timescaledb.default_hypercore_use_access_method;
+show timescaledb.default_hypercore_use_access_method;
+ timescaledb.default_hypercore_use_access_method 
+-------------------------------------------------
+ none
+(1 row)
+
+-- Using GUC should compress using the hyperstore
+set timescaledb.default_hypercore_use_access_method to on;
```

Comment:
Shouldn't we test behavior also under `off` and `none`? I mean caliing  `compress_chunk()` when set to off or none? (I guess none is implicitly tested by other tests, or?)

Reply:
Both "none" and "off" are tested by other tests, but I can extend these with some more tests.